static/js/controllers/reports.js

```diff
@@ -40,24 +40,43 @@ var _ = require('underscore'),
 
       var liveList = LiveList.reports;
 
+      var recordReadStatus = function(models) {
+        var ids = models.map(function(model) {
+          return 'read:report:' + model._id;
+        });
+        return DB({ meta: true })
+          .allDocs({ keys: ids })
+          .then(function(response) {
+            for (var i = 0; i < models.length; i++) {
+              models[i].read = !!response.rows[i].value;
+            }
+          });
+      };
+
       var updateLiveList = function(updated) {
-        _.each(updated, function(report) {
-          liveList.update(report);
+        recordReadStatus(updated).then(function() {
+          updated.forEach(function(report) {
+            liveList.update(report);
+          });
+          $scope.hasReports = liveList.count() > 0;
+          liveList.refresh();
+          if ($state.params.id) {
+            liveList.setSelected($state.params.id);
+          }
         });
-        $scope.hasReports = liveList.count() > 0;
-        liveList.refresh();
-        if ($state.params.id) {
-          liveList.setSelected($state.params.id);
-        }
       };
 
       var setSelected = function(model) {
         $scope.setSelected(model);
         if ($scope.selectMode) {
           return;
         }
-        if (!$scope.isRead(model.doc)) {
+        var listModel = _.findWhere(liveList.getList(), { _id: model._id });
+        if (listModel && !listModel.read) {
           $scope.readStatus.forms--;
```

Comment:
Also it's actually an "unread" count, not a "read" one... Renamed to `unreadCount`.